js/controllers/menu.js

```diff
@@ -17,19 +17,33 @@ function MenuController(tabs) {
   $(document).bind('tabsave', this.onTabSave.bind(this));
 }
 
-MenuController.prototype.onNewTab = function(e, tab) {
-  var id = tab.getId();
-  var name = tab.getName();
-  var listItem = $('<li id="tab' + id + '" draggable="true">' +
```

Comment:
Out of interest, why such a big change? I don't mind using `document.createElement` and all that but I'm curious as to why you needed to make this change rather than mostly keeping what was already here.

Reply:
Yeah mostly I wanted to escape the filename since it's user input. So I could either rewrite it in jQuery or es6. I figured moving away from jQuery was better. I've been roughly updating things as I touch them. Do you think it's worth doing that in general?

Reply:
I think it's a bit of a principle thing. I don't mind a piecemeal update where you slowly pull out jQuery, and if you think relying less on jQuery is the way to go that sounds good to me. Generally I prefer using the DOM directly so this sounds like a good direction to go in :)